drivers/ethernet/eth_stm32_hal_priv.h

```diff
@@ -36,6 +37,11 @@ extern const struct device *eth_stm32_phy_dev;
 #elif defined(CONFIG_SOC_SERIES_STM32H7X) || defined(CONFIG_SOC_SERIES_STM32H7RSX)
 #define __eth_stm32_desc __attribute__((section(".eth_stm32_desc")))
 #define __eth_stm32_buf  __attribute__((section(".eth_stm32_buf")))
+#elif defined(CONFIG_SOC_SERIES_STM32MP13X)
+#define ETH_DMA_REGION  DT_INST_PHANDLE(0, memory_regions)
+#define ETH_DMA_LINKER_REGION_NAME LINKER_DT_NODE_REGION_NAME(ETH_DMA_REGION)
+#define __eth_stm32_desc ALIGN_32BYTES(__attribute__((__section__(ETH_DMA_LINKER_REGION_NAME))))
+#define __eth_stm32_buf  ALIGN_32BYTES(__attribute__((__section__(ETH_DMA_LINKER_REGION_NAME))))
```

Comment:
Two notes on this, none are blocking:
-Use of `ALIGN_32BYTES`. Is that required here or only for `dma_rx_desc_tab` as done for STM32N6 ? Can't it be replaced by `__aligned(32)` to harmonize code ?
-Use of DT region is interesting and maybe can replace existing churn on STM32H7/H7R/S, so we avoid having multiple implementations of the same thing. @juickar Can you log this last point for DV4.4.0 ?